server/dash/dashboards/neighborhood_recent.py

```diff
@@ -186,7 +171,12 @@ def update_text(selected_council):
 def update_figure(selected_council):
     figure_df = df.query(f"councilName == '{selected_council}' and createdDate >= '{start_date}'").groupby(['createdDate', 'typeName'])['srnumber'].count().reset_index()  # noqa
     figure_df.typeName = figure_df.typeName.map(lambda x: '<br>'.join(textwrap.wrap(x, width=16)))  # noqa
```

Comment:
Do we want this styling applied when there is no data as well? It seems like it wouldn't be applied since the we are changing the object that figure_df references.

Reply:
Move the line to after the checking condition. Thanks for the catch!